### PR TITLE
patch: activities page images support

### PR DESCRIPTION
**Pull request by MoMentalochka**
- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] The PR has self-explained commits history.

- [x] The code is mine or it's from somewhere with an Apache-2.0 compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.
- [x] If the code introduces new classes, methods, I provide a valid use case for all of them.

## Changes:

- Images support
- Update filters

### CODE REVIEW

File: components/comments/templates/post.html [hunk 1]
```diff
@@ -88,7 +88,7 @@
 
 					<div class="leaveComment" placeholder="<%- placeholder || dplace %>"></div>
 					
-					<% if(embedimage) {%>
+					<% if(true) {%>
```
Comment (shpingalet007) [inline components/comments/templates/post.html:91]: Why not just removing this conditional construction @MoMentalochka?

**Review by shpingalet007 — CHANGES_REQUESTED** [review]

File: components/comments/templates/post.html (hunk 1 above)
Comment (MoMentalochka) [reply components/comments/templates/post.html:91]: miss play detected

Comment (shpingalet007) [reply components/comments/templates/post.html]: Fore reference: this conditional construction was changed for tests only. Changes are made, can proceed with this...

**Review by shpingalet007 — APPROVED** [review]
Quality checked

**Pull request merged**